src/pages/Portfolio/LendingStats/BadgesList.vue

```diff
@@ -16,43 +16,47 @@
 		</div>
 	</div>
 	<!-- Badges Template -->
-	<div class="tw-grid tw-grid-cols-1 md:tw-grid-cols-2 tw-gap-2" v-else>
+	<div
+		v-else
+		class="tw-w-full tw-inline-flex tw-flex-wrap tw-justify-center lg:tw-justify-start tw-gap-5 md:tw-gap-4
+			lg:tw-gap-6 tw-pl-2 lg:tw-pl-5"
+	>
 		<p v-if="completedAchievements.length === 0">
 			You haven't earned any badges yet.
 		</p>
-		<div
-			v-else
-			v-for="badge in completedAchievements"
+		<BadgeCard
```

Comment:
We couldn't re-use the component from the public profile work?

Reply:
Both grids work slightly different, but we are reusing `BadgeCard` as well...